task-management/src/main/webapp/vue-app/taskCommentsDrawer/components/TaskLastComment.vue

```diff
@@ -21,6 +21,7 @@
     <div class="commentHeader d-flex">
       <exo-user-avatar 
         :profile-id="comment.author.username"
+        :key="comment.author.username"
```

Comment:
`        :key="comment.id"`
The `key` must be really unique, if we have the same user who has commented multiple times, the key will be redundant, thus we should use the id of the comment to be ensure the uniqueness of this bloc.

Reply:
@Montassar-Hamadi